test/inductor/test_torchinductor_opinfo.py

```diff
@@ -369,6 +369,8 @@ def wrapper_noop_set_seed(op, *args, **kwargs):
     },
     ("std_mean.unbiased", "cuda", f16): {"reference_in_float": True},
     ("uniform", "cuda"): {"reference_in_float": True},
+    ("_unsafe_masked_index_put_accumulate", "cuda", f16): {"atol": 1e-4, "rtol": 0.01},
+    ("_unsafe_masked_index_put_accumulate", "cpu", f16): {"atol": 1e-4, "rtol": 0.01},
```

Comment:
why don't we have hte same in `index_put`?

Reply:
I think it's because the test input samples are slightly different.